### ISSUE TITLE
1.7.4 doesn't appear to update posts when importing from github

### ISSUE THREAD

**Issue by dirkhh** [mAAdhaTTah/wordpress-github-sync#157]
When I click "Import from github" I get forwarded to a page that doesn't exist (wp-admin/options-general.php?page=wordpress-github-sync instead of wp-admin/options-general.php?page=wp-github-sync), but more annoyingly, posts aren't actually updated.

save_posts() does this which I don't understand:

```
                foreach ( $posts as $post ) {
                        $args    = apply_filters( 'wpghs_pre_import_args', $post->get_args(), $post );

                        remove_filter('content_save_pre', 'wp_filter_post_kses');
                        $post_id = $post->is_new() ?
                                wp_insert_post( $args, true ) :
                                wp_update_post( $args, true );
```
if instead I call
`    wp_update_post($post->post, true);`
it actually works, but I find it hard to believe that that's the correct solution - so clearly I must be missing something...

**Comment by mAAdhaTTah** [comment]
Well, the first bug (page slug) was fixed by #155; I just have to cut a release, which I'll handle this morning. Let me do that, update the plugin, then let me know if the issue still exists.

**Comment by mAAdhaTTah** [comment]
1.7.5 is pubbed, update to that version and let me know if you still have issues. If you've been modifying the plugin files, you'll want to backup your changes before doing so.

**Comment by dirkhh** [comment]
So yes, the first bug is fixed with 1.7.5.
Right now I cannot reproduce the second bug, so I'll close this issue.
Thanks for all the prompt help!
